modules/adding-ca-certs-to-config.adoc

```diff
@@ -0,0 +1,110 @@
+[id="adding-ca-certs-to-config"]
+= Adding additional Certificate Authorities to {productname-ocp}
+
+The following example shows you how to add additional Certificate Authorities to your {productname-ocp} deployment.
+
+.Prerequisites
+
+* You have base64 decoded the original config bundle into a `config.yaml` file.
+* You have a Certificate Authority (CA) file or files.
+
+.Procedure
+
+. Create a new YAML file, for example, `extra-ca-certificate-config-bundle-secret.yaml`:
++
+[source,terminal]
+----
+$ touch extra-ca-certificate-config-bundle-secret.yaml
+----
+
+. Create the `extra-ca-certificate-config-bundle-secret` resource.
+
+.. Create the resource by entering the following command:
++
+[source,terminal]
+----
+$ oc -n <namespace> create secret generic extra-ca-certificate-config-bundle-secret \
+  --from-file=config.yaml=</path/to/config.yaml> \ <1>
+  --from-file=ssl.cert=</path/to/ssl.cert> \ <2>
```

Comment:
@stevsmit if this doc targets to `Adding additional Certificate Authorities to quay, which is third party external services like LDAP, OIDC, and storage systems`, then the ssl.cert, ssl.key is not necessary. Same to below Standalone Quay part. 
The SSL/TLS is for Quay is to utilize a self-signed certificate for Quay server itself, like in [poc doc](https://docs.redhat.com/en/documentation/red_hat_quay/3.12/html-single/proof_of_concept_-_deploying_red_hat_quay/index#creating-a-certificate-authority). 
`extra_ca_certs`  is a separate topic, pls refer to our [previous discussion in slack](https://redhat-internal.slack.com/archives/C0428JVU8MS/p1724679750386569) if still not clear. 
And of course, if user want to use they together like you posted, it's a valid scenario.